solidity/test/BancorNetwork.js

```diff
@@ -122,6 +125,14 @@ async function initTokensAndConverters(accounts) {
     bancorNetwork = await BancorNetwork.new(contractRegistry.address);
     await contractRegistry.registerAddress(ContractRegistryClient.BANCOR_NETWORK, bancorNetwork.address);
 
+    let converterRegistry = await BancorConverterRegistry.new(contractRegistry.address);
+    let converterRegistryData = await BancorConverterRegistryData.new(contractRegistry.address);
+    await contractRegistry.registerAddress(ContractRegistryClient.BANCOR_CONVERTER_REGISTRY, converterRegistry.address);
+    await contractRegistry.registerAddress(ContractRegistryClient.BANCOR_CONVERTER_REGISTRY_DATA, converterRegistryData.address);
+
+    let pathFinder = await BancorNetworkPathFinder.new(contractRegistry.address);
+    await contractRegistry.registerAddress(ContractRegistryClient.CONVERSION_PATH_FINDER, pathFinder.address);
+
     bnt = await ERC20Token.new('BNT', 'BNT', 2, 10000000);
```

Comment:
@yudilevi : I recommend renaming this to `bntToken`.

Reply:
it's a bit redundant since bnt is Bancor Network Token but sure, will update.

Reply:
done